README.md

```diff
@@ -75,11 +75,13 @@ _You may need to install libgit2. In case you are in mac just use `brew install
 
 ## Setup
 
-### Getting your .so files
+### Getting the files
 
 From onwards tree-hugger 0.9 we ship a new command `download_libs`.
 
-If you are working on Debian based Linux or Newer version of MacOS then you should probably just use this command to get the library. At any point of time we will maintain a .so file for both those OSs with all the supported languages in it. 
+If you are working on Debian based Linux or Newer version of MacOS then you should probably just use this command to get the library. At any point of time we will maintain a [.so](https://en.wikipedia.org/wiki/Static_library) file for both those OSs with all the supported languages in it.
+
+If you are working on Windows then you would be getting a [.dll](https://en.wikipedia.org/wiki/Dynamic-link_library) file rather than a .so file.
```

Comment:
Maybe we should warn users here about the 32 bit thing and that it may cause a memory overflow. So that they are aware of this issue. (Also if you can put the param-link of issue from tree-sitter then it would be great.

Reply:
Okay. I would add it here.